src/index.js

```diff
@@ -32,28 +32,34 @@ class TelldusTDToolPlatform {
     this.log = log
     this.config = config
     this.homebridge = homebridge
+    this.senzors = config["sensors"] || [];
   }
 
   accessories(callback) {
     this.log('Loading devices...')
     TDtool.listDevices().then(deviceCandidates => {
       const devices = deviceCandidates.filter(d => d.type === 'device')
-      this.log(foundOfTypeString('device', devices.length))
+      this.log(foundOfTypeString('device', devices.length, 'tdtool --list-devices'))
       return devices
     }).then(devices => {
-      return TDtool.listSensors().then(sensors => {
-        this.log(foundOfTypeString('sensor', sensors.length))
-        sensors.forEach((current, index) => {sensors[index].name = `Thermometer ${current.id}`})
-        return devices.concat(sensors)
-      })
+      if (this.senzors.length > 0) {
+        this.log(foundOfTypeString('sensor', this.senzors.length, 'config.json'))
```

Comment:
As this is user input, we should verify that the fields supplied are `'name'` (optional), `'type'` and `'id'` and nothing else.